lib/playwright/javascript/value_serializer.rb

```diff
@@ -24,7 +24,7 @@ def serialize
           @handles << value.channel
           { h: index }
         when nil
-          { v: 'undefined' }
+          { v: 'null' }
```

Comment:
What would happen without this change?

Reply:
```ruby
    it "should work with pass null" do
      with_page do |page|
        page.set_content("<div></div>")
        page.eval_on_selector("div", "e => e.foo = null")
        locator = page.locator("div")
        expect(locator).to have_js_property("foo", nil)
      end
    end
```
This test case will fail because it expects null